ui/forms.md

```diff
@@ -1,256 +1,13 @@
 ## Forms
 
-This attempts to document the state of forms in ui-classic, including all the stages of conversion, from pure rails to Angular.
+The recommended way of writing ManageIQ UI forms is using [Data Driven Forms](http://data-driven-forms.surge.sh/).
 
-Each stage's description should contain:
+Historically, ManageIQ UI still has rails forms and angular forms, for information about those, please see [the wiki](https://github.com/ManageIQ/manageiq-ui-classic/wiki/Forms-%28kinds%29).
 
-  * the main characteristics of such a form with pointers where to look
-  * the list of actions involved during the form's lifecycle
-  * a code example of a minimal form with
-    * one text input validated for length
-    * one select, the choice in which updates a useless message div
-    * the default form buttons
-  * a recommended migration path
+---
 
+Manageiq UI imports data driven forms and exposes a `@@ddf` webpack alias for both local and plugin imports.
```

Comment:
Would be nice to have a little bit more, in a form of example maybe:
```
import MiqFormRenderer, { componentTypes, validatorTypes } from '@@ddf';
```

The `MiqFormRenderer` is our wrapper around DDF with our custom components and implicitly translated button labels. For more information see the definition of the component (should have a link to data-driven-form.jsx). The `componentTypes` and `validatorTypes` are a set of constants provided by DDF for creating a schema.